src/sgame/sg_maprotation.cpp

```diff
@@ -1042,6 +1042,14 @@ Send commands to the server to actually change the map
 static void G_IssueMapChange( int index, int rotation )
 {
 	mrNode_t *node = mapRotations.rotations[ rotation ].nodes[ index ];
+  mrCondition_t *condition;
+
+  if( node->type == NT_CONDITION )
+  {
+    condition = &node->u.condition;
+    node = condition->target;
+  }
+
```

Comment:
interesting, last time I tried to only remove the `|| conditonal` test I got random ugly crashes, maybe I was missing this.

Reply:
yeah the rotation will try to load the node with the condition as map, which results in random letters as mapname